src/SFML/Graphics/GLExtensions.cpp

```diff
@@ -52,6 +52,38 @@
 
 namespace sf::priv
 {
+////////////////////////////////////////////////////////////
+void extensionSanityCheck()
+{
+    static const auto check = [](int& flag, auto... entryPoints)
```

Comment:
The `XXX_dependencies` macros aro only used in this file, right? Can we define them in this file rather than the `.hpp` to avoid leaking more macros? It would also make it more obvious why `entryPoints` is variadic.

Reply:
The idea is that they are defined right under the definition of the extension functionality so in case it has to be adjusted (add/remove functions) we would only have to make changes to 1 place (that fits on a single screen) instead of 2. Splitting them leads to a higher risk that something is accidentally left out. Also, `gl.h` already defines 1000s of macros, so us adding a handful more with carefully chosen prefixes shouldn't really matter that much.